spec/models/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    author = User.create(name: 'author', photo: 'photo', bio: 'bio', posts_counter: 2)
+    post = Post.create(title: 'title', text: 'text', comments_counter: 2, author_id: author.id, likes_counter: 2)
+    subject { Like.new(author_id: 1, post_id: 1) }
+
+    before { subject.save }
+
+    it 'author id should be an integer' do
+      subject.author_id = 'string'
+      expect(subject).to_not be_valid
+    end
+
+    it 'post id should be an integer' do
+      subject.post_id = 'string'
+      expect(subject).to_not be_valid
+    end
+
+    it 'updates the likes counter' do
+      expect(post.likes_counter).to eq(2)
+    end
+  end
```

Comment:
- [ ] Please make sure all your test are passing :pray: As you can see below you have some failing tests :point_down: 
  ![Screenshot from 2023-04-06 23-52-45](https://user-images.githubusercontent.com/92303891/230514265-d71e600b-9b95-46eb-a6fc-dc517a3aee2f.png)

Reply:
Hello @Owusu-Desmond  thank you to review my project, but when I run the test on my local machine it's still all tests are passing are you sure you checked my project & no changes are done there?
you can look below:
![image](https://user-images.githubusercontent.com/104406349/230544923-d8b09e86-926e-438a-a544-3c98b93ea6bc.png)